Added Issue and Pull Request GitHub templates. (#8)
LucaPipolo/gulp-static-starterkit: .github/PULL_REQUEST_TEMPLATE.md

## Checklist for this Pull Request

> - [ ] Check if there is already an open issue. If not, please create one.

Reviewer: ```suggestion
- [ ] Check if a related or similar issue already exists. If not, please create one.
```

> - [ ] Check the added code or even all commits' message styles matches our requested structure.

Reviewer: ```suggestion
- [ ] Check if the added code and commits messages match our requested style and structure.
```

> - [ ] Check your code additions will not fail code linting checks.

Reviewer: ```suggestion
- [ ] Check that your code additions will not fail code linting checks.
```

> - [ ] Check that package fully work if installed into a sample static website.

Reviewer: ```suggestion
- [ ] Check that the package fully works if installed into a sample static website.
```
